db/migrate/20150406192448_create_active_admin_comments.rb

```diff
@@ -0,0 +1,19 @@
+class CreateActiveAdminComments < ActiveRecord::Migration
+  def self.up
+    create_table :active_admin_comments do |t|
+      t.string :namespace
+      t.text   :body
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.